airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/JdbcBufferedConsumerFactory.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2020 Airbyte
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package io.airbyte.integrations.destination.jdbc;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.common.base.Preconditions;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.commons.text.Names;
+import io.airbyte.integrations.base.DestinationConsumer;
+import io.airbyte.integrations.destination.NamingConventionTransformer;
+import io.airbyte.integrations.destination.WriteConfig;
+import io.airbyte.integrations.destination.buffered_stream_consumer.BufferedStreamConsumer;
+import io.airbyte.integrations.destination.buffered_stream_consumer.BufferedStreamConsumer.OnCloseFunction;
+import io.airbyte.integrations.destination.buffered_stream_consumer.BufferedStreamConsumer.OnStartFunction;
+import io.airbyte.integrations.destination.buffered_stream_consumer.BufferedStreamConsumer.RecordWriter;
+import io.airbyte.protocol.models.AirbyteMessage;
+import io.airbyte.protocol.models.ConfiguredAirbyteCatalog;
+import io.airbyte.protocol.models.SyncMode;
+import java.time.Instant;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+// Strategy:
```

Comment:
can you use 
```
/*
 * 
 */
``` 
comment style instead of // ?

// should be use for one liners

Reply:
Unfortunately our formatter turns this into soup.
```
/*
  * Strategy:
  * 1. Create a temporary table for each stream
  * 2. Accumulate records in a buffer. One buffer per stream.
  * 3. As records accumulate write them in batch to the database. We set a minimum numbers of records
  * before writing to avoid wasteful record-wise writes.
  * 4. Once all records have been written to buffer, flush the buffer and write any remaining records
  * to the database (regardless of how few are left).
  * 5. In a single transaction, delete the target tables if they exist and rename the temp tables to
  * the final table name.
 */
```
becomes
```
/*
 * Strategy: 1. Create a temporary table for each stream 2. Accumulate records in a buffer. One
 * buffer per stream. 3. As records accumulate write them in batch to the database. We set a minimum
 * numbers of records before writing to avoid wasteful record-wise writes. 4. Once all records have
 * been written to buffer, flush the buffer and write any remaining records to the database
 * (regardless of how few are left). 5. In a single transaction, delete the target tables if they
 * exist and rename the temp tables to the final table name.
 */
```

after we run `.gradlew format`. The way around it is to add the `<p>` html tags, but while retaining the line break this makes it similarly unreadable imho. if you prefer the `<p>` tags though i can do them. You can see an example of what the `<p>` tags look like [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryDestination.java#L191-L212).

Reply:
and if you make it a javadoc? (`/**`) ?

Reply:
same issue.